appveyor.yml

```diff
@@ -17,6 +17,11 @@ install:
     - rustup-init.exe -y --default-host %TARGET% --default-toolchain nightly
     - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin;C:\Users\appveyor\.rustup\toolchains\nightly-%TARGET%\bin
     - if defined MSYS2_BITS set PATH=%PATH%;C:\msys64\mingw%MSYS2_BITS%\bin
+    - cargo install rustup-toolchain-install-master || echo "rustup-toolchain-install-master already installed"
+    - set RUSTC_HEAD=`git ls-remote https://github.com/rust-lang/rust.git HEAD | tr -d ",." | tr " \t" "\n" | grep -e "HEAD" -v`
```

Comment:
let's use `rustup-toolchain-install-master -n master -f` from https://github.com/kennytm/rustup-toolchain-install-master/pull/6 when it merges

though I'm okay with merging this first and switching over later